lib/cyclone-server/src/server.rs

```diff
@@ -103,6 +109,41 @@ impl Server<(), ()> {
             wrong @ IncomingStream::HTTPSocket(_) => {
                 Err(ServerError::WrongIncomingStream("http", wrong.clone()))
             }
+            wrong @ IncomingStream::VsockSocket(_) => {
+                Err(ServerError::WrongIncomingStream("http", wrong.clone()))
```

Comment:
Shouldnt the "http" actually be "uds"? I know you just copied the line above, but I think it's wrong. No big deal, big if you could fix them it would be great.

The same applies to the vsock branch that got added. The failure should report "vsock" not "http"